logger/logger.go

```diff
@@ -0,0 +1,23 @@
+// Package config defines the Cloudinary configuration.
+package logger
+
+import (
+	"log"
+)
+
+// Logger cloudinary logger
+type LoggerFunction func(...interface{})
+
+type Logger struct {
+	ErrorLog LoggerFunction
+	DebugLog LoggerFunction
+	FatalLog LoggerFunction
+}
+
+func New() *Logger {
+	return &Logger{
+		ErrorLog: log.Println,
```

Comment:
What if user is already using logger like `logrus`, it would be better to leverage that and use `log.Error`, `log,Debug` etc. It would also preserve formatting, hooks, etc.

Reply:
And our end-user is free to override this functions with anything else during configuration.
`examples/example.go` contains the example of it.
To use logrus our user can just pass there a function from logrus (if it's support LoggerFunction interface) or pass a function-wrapper.

Having this our library is agnostic for any logger you want to use.